tests/app/tests/test_webpack.py

```diff
@@ -219,7 +217,7 @@ def test_request_blocking(self):
         # FIXME: This will work 99% time but there is no garauntee with the
         # 4 second thing. Need a better way to detect if request was blocked on
         # not.
-        wait_for = 3
+        wait_for = 4
```

Comment:
Cool! I suppose this change is to match the comment. Can I suggest to fix the garauntee typo (that should be guarantee)?

Reply:
Sure, I'll do it.